docs/shop/companysettings/orderlimits.rst

```diff
@@ -0,0 +1,30 @@
+Order Ceiling
+=============
+
+The order ceiling defines an order limit for users per month. It can only be set
```

Comment:
The order ceiling defines the maximum amount of money, a user is allowed to order(spend) per month.